tests/bot/exts/moderation/infraction/test_infractions.py

```diff
@@ -206,7 +206,7 @@ async def test_voice_unban_user_found(self, format_user_mock, notify_pardon_mock
         notify_pardon_mock.return_value = True
         format_user_mock.return_value = "my-user"
 
-        result = await self.cog.pardon_voice_ban(self.user.id, self.guild, "foobar")
+        result = await self.cog.pardon_voice_ban(self.user.id, self.guild)
         self.assertEqual(result, {
             "Member": "my-user",
             "DM": "Sent"
```

Comment:
Won't this fail because you don't specify `notify`?

Reply:
No. This function has a default value for that parameter. Furthermore, tests run in CI and you can see CI is passing.